code/datums/wires/wires.dm

```diff
@@ -87,8 +87,9 @@ var/list/wireColours = list("red", "blue", "green", "black", "orange", "brown",
 	if(ishuman(user))
 		var/mob/living/carbon/human/H = user
 		var/obj/item/organ/internal/eyes/eyes = H.get_int_organ(/obj/item/organ/internal/eyes)
-		if(eyes && H.disabilities & COLOURBLIND)
-			replace_colours = eyes.replace_colours
+		if(eyes && (H.disabilities & COLOURBLIND || eyes.dark_view >= 6))
```

Comment:
Maybe this can use `EYE_SHINE_THRESHOLD` instead of 6 as it's a define (with the same value currently) and is used for similar darksight-based behaviours :thinking:

Reply:
Good catch.